### PR TITLE
Add news for TC39 in July 2021

### PR DESCRIPTION
**Pull request by codehag**
cc @iainireland

### CODE REVIEW

File: _posts/2021-08-09-tc39.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+### Array filtering
+* [**Notes**](https://github.com/tc39/notes/blob/master/meetings/2021-07/july-14.md#array-filtering--grouping-for-stage-2)
+* [**Proposal Link**](https://github.com/tc39/proposal-array-filtering)
+* [**Slides**](https://docs.google.com/presentation/d/1fY_jsD8bVZ8P95Mr7cEr3WdCbhMLdEQ7OS5hhLCbfJ4/edit)
+* **Summary**: This proposal was two proposals bundled. It introduces a .filterOut method which is
+    an alias for a negative filter, such as `[1, 2, 3].filter(x => !(x > 2))` which would return all
+    of the elements less than or equal to 2. This did not move
+    forward. A second proposal, `groupBy`, which groups elements related to a condition (for
+    example, `[1,2,3].groupBy(x => x > 2)`, would return `[[1,2],[3]]`); GroupBy advanced to stage
+    1.
```
Comment (jridgewell) [inline _posts/2021-08-09-tc39.md]:
```suggestion
* **Summary**: This proposal was two proposals bundled. It introduces a .filterReject method which is
    an alias for a negative filter, such as `[1, 2, 3].filter(x => !(x > 2))` which would return all
    of the elements less than or equal to 2. This did not move
    forward. A second proposal, `groupBy`, which groups elements related to a condition (for
    example, `[1,2,3].groupBy(x => x > 2)`, would return `{false:[1,2],true: [3]}`); GroupBy advanced to stage
    1 as a separate [proposal](https://github.com/tc39/proposal-array-grouping).
```

**Review by jridgewell — APPROVED** [review]

**Comment by codehag** [comment]
Iain is on PTO... so I will merge this after an alternative team review, i think we are all foggy on what happened that meeting at this point anyway.

File: _posts/2021-08-09-tc39.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+
+
+## Proposals Seeking Advancement to Stage 4
+
+### Ergonomic Brand Checks
+* [**Notes**](https://github.com/tc39/notes/blob/master/meetings/2021-07/july-14.md#ergonomic-brand-checks-for-stage-4)
+* [**Proposal**](https://github.com/tc39/proposal-private-fields-in-in)
+* [**PR**](https://github.com/tc39/ecma262/pull/2290)
+* [**Spec**](https://tc39.es/proposal-private-fields-in-in/)
+* **Summary**: Provides an ergonomic way to check the presense of a private field when one of its
```
Comment (jandem) [inline _posts/2021-08-09-tc39.md]: Nit: presence, typo

**Review by jandem — APPROVED** [review]
Looks great, thanks for writing this up.

File: _posts/2021-08-09-tc39.md (hunk 2 above)
Comment (codehag) [inline _posts/2021-08-09-tc39.md]:
```suggestion
* **Summary**: Provides an ergonomic way to check the presence of a private field when one of its
```

**Pull request merged**